### DISCUSSION TITLE
How many scripts could run at the same time ?

### DISCUSSION THREAD

**Discussion by CapCatBite** [HotNoob/Oracle-Free-Arm-VPS-PS#9] [Q&A]
if i used this on multiple accounts, multiple regions, is it possible in the first place ? 3 or 4 accounts on 4 scripts different regions/servers ?

**Comment by HotNoob** [comment]
https://github.com/HotNoob/Oracle-Free-Arm-VPS-PS/blob/b28259f1f5f5f2f48cbeff85c2ac3bbd840190db/create_oracle_arm_instance.ps1#L38-L39

edit theese two lines, and run as many as you want. 
you just have to create profiles for each account. 

never tried different regions with the same account, but should work. just make a copy, edit and run.

**Comment by CapCatBite** [comment]
ok, 2 questions, i am literally a newwb in coding quite literally, my apologize.

1st
$configFile = $env:USERPROFILE+"\.oci\config" <<  i understood what to edit in the profile line 38, but line 39, edit what in it ?
 
2nd
so multiple profiles with request on the same pc is it possible without causing a time out or ban or any of such ?

**Comment by HotNoob** [comment]
$configFile is the location of your oci config. the script uses the default location. depending on how you setup your new profiles, it can change.

https://docs.oracle.com/en-us/iaas/Content/Functions/Tasks/functionsconfigureocicli.htm

2. you might have to adjust the delays; the script does automatically increase the delay as it runs into "too frequent" errors. 
as for banning, i don't think so, as they allow multiple profiles. 
oci cli is designed to be script accessible.

it is of course possible they enforce something, but i don't think 4 would trigger anything.

**Comment by CapCatBite** [comment]
alright then, i will make 4 profiles, give it a try and keep you updated if anything came up, thank you a ton for the quick response dude, you rock ~
